pubpub/pubpub
# dev(ts-migrate): move config.js to repo root (1/3)

This PR moves the config.js file to the repo root. The `server` directory will now contain only the TypeScript source code of the API application, and the config file will remain un-transpiled at the root of the project.

_Test Plan_
* Copy your existing local config.js file into the repo root.
* Start PubPub and make sure the app loads and functions properly!
